hibernate-core/src/test/java/org/hibernate/orm/test/jpa/orphan/onetomany/DeleteSharedOneToManyOrphansTest.java

```diff
@@ -78,7 +78,6 @@ public void cleanupTest(EntityManagerFactoryScope scope) throws Exception {
 
 	@Test
 	@TestForIssue(jiraKey = "HHH-11144")
-	@FailureExpected( jiraKey = "HHH-11144" )
```

Comment:
It is such a wonderful feeling to get rid of such stuff, :).

Reply:
unfortunately it was solved but because a wrong side effect introduced by the `seize` method i introduced, after removing the method it the issue  is back :(